### DISCUSSION TITLE
Redesign multiplayer room cards to have better emphasis on if the room is locked or open

### DISCUSSION THREAD

**Discussion by LukynkaCZE** [ppy/osu#19663] [Ideas]
I think the biggest issue I have currently with them that its not very clear if the room is open or locked. The little lock in the top right corner isn't really intuitive as the first place you look is the text on left

I suppose having a open/locked lock on the left instead of the little bit of beatmap background that there is currently

Here is little design I threw together
![image](https://user-images.githubusercontent.com/48604271/183598920-5f258651-c7f6-4495-bcd1-bcf78d8527eb.png)

**Comment by peppy** [comment]
Rather than changing the design, locked rooms should be sorted to the bottom, not shown by default, and grouped together.

99% of the time users will not want to see these interspersed between public rooms.

**Reply by LukynkaCZE** [reply]
Yep that is also really good and way easier solution! Though if the current plan is to redesign the "entire" UI over time, you might as well do something like this when you get to the multiplayer lounge
